subprojects/gradle/module-dependencies-graph/src/main/kotlin/com/avito/module/metrics/metrics/Matrix.kt

```diff
@@ -0,0 +1,25 @@
+package com.avito.module.metrics.metrics
+
+internal class Matrix<KEY, DATA> {
```

Comment:
I propose to rename KEY to COORDINATE
because it's a Matrix. Matrix should have two coordinate and value

Reply:
Renamed